TicTacToeApp/ViewController.swift

```diff
@@ -1,152 +1,99 @@
 
 import UIKit
 
-public class BoardButton: UIButton {
-}
-
-public class BoardView: UIView {
-
-  // hypothetical interface
-  public func enableInput() {}
-  public func disableInput() {}
-  public func clear() {}
-
-}
-
-// Refactoring alert dialog
-// Might be overkill. Let's revisit after board button refactorings
-//public class ConfirmResetGameAlert: UIAlertController {}
-// Alternatively, consider builder patter, or just wrapping the alert dialog
-//     i.e. composition over inheritance
-//public class ConfirmResetGameAlert {
-//  var alertController: UIAlertController?
-//}
-
 public class ViewController: UIViewController {
 
-  private var game = Game()
-  private let gamePrompt = GamePrompt()
+  @IBOutlet public weak var prompt: UILabel!
+  @IBOutlet public var boardButtons: [UIButton]!
+  @IBOutlet public weak var resetButton: UIButton!
+  @IBOutlet var gameModeButtons: [UIButton]!
+
+  private var gameState = Game()
+  private var gameBoard: BoardButtons?
+  private var gamePrompt: GamePrompt?
   public var currentMode = GameMode.HumanVsHuman
```

Comment:
We've been thinking of doing that - we'll also definitely need to revisit GameMode in the future. We'll try to integrate it into Game when the time comes.